clang/lib/DPCT/RulesAsm/AsmMigration.cpp

```diff
@@ -536,6 +536,7 @@ bool SYCLGenBase::emitBuiltinType(const InlineAsmBuiltinType *T) {
   case InlineAsmBuiltinType::u16x2:  OS() << MapNames::getClNamespace() + "ushort2"; break;
   case InlineAsmBuiltinType::bf16:   OS() << MapNames::getClNamespace() + "ext::oneapi::bfloat16"; break;
   case InlineAsmBuiltinType::f16x2:  OS() << MapNames::getClNamespace() + "half2"; break;
+  case InlineAsmBuiltinType::voidType:  OS() << "void"; break;
```

Comment:
voidType is not needed here, pls remove line here.

Reply:
Removed it.

Using **byte** type now as per your suggestion.